auto_submit/lib/requests/github_webhook.dart

```diff
@@ -39,9 +43,9 @@ class GithubWebhook extends RequestHandler {
     print('pullRequest: ${pullRequest.id}');
 
     if (hasAutosubmit) {
-      log.info(body);
+      log.info('Find one that has autosubmit label $body');
```

Comment:
nit: Logs are more helpful/should be in past tense.

`Find` -> `Found pull request with auto submit label`

Reply:
Gotcha Casey, done.